recipes/glpk/all/conanfile.py

```diff
@@ -0,0 +1,151 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conan.tools.files import rename
+from contextlib import contextmanager
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class GlpkConan(ConanFile):
+    name = "glpk"
+    description = "GNU Linear Programming Kit"
+    homepage = "https://www.gnu.org/software/glpk"
+    topics = ("linear", "programming", "simplex", "solver")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "GPL-3.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _is_msvc(self):
+        return str(self.settings.compiler) in ["Visual Studio", "msvc"]
```

Comment:
```suggestion
```

It's internalized since Conan 1.45 https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc